salt/modules/boto_dynamodb.py

```diff
@@ -252,6 +252,25 @@ def update_global_secondary_index(table_name, global_indexes, region=None,
     return table.update_global_secondary_index(global_indexes)
 
 
+def list_tables(region=None, key=None, keyid=None, profile=None):
+    '''
+    List all DynamoDB tables visible to the supplied credentials.
+
+    CLI example::
+
+        salt myminion boto_dynamodb.list_tables region=us-east-1
+    '''
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+    estn = ''
+    tables = []
+    while estn is not None:
+        args = {'exclusive_start_table_name': estn} if estn else {}
+        r = conn.list_tables(**args)
```

Comment:
I know there are other references to single letter variables in the `boto_` files, but our style guide declares that we should be using variables of at least three letters. Would you mind changing those references in this new function? That way we can at least have this addition on the right track. :)